src/CONST.ts

```diff
@@ -6816,6 +6816,14 @@ const CONST = {
         },
     },
 
+    DEVICE_CONTACT: {
```

Comment:
I feel like this is duplicated from `modules/ExpensifyNitroUtils/specs/ContactsModule.nitro.ts` - could we just use that as the source of truth?